### PR TITLE
Update pktmon.md

### PR DESCRIPTION
**Pull request by george-guirguis**
updated the functionality since a new version of the tool is out already

### CODE REVIEW

**Comment by PRMerger9** [comment]
@george-guirguis : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: WindowsServerDocs/networking/technologies/pktmon/pktmon.md [hunk 1]
```diff
@@ -50,29 +50,14 @@ Use these best practices for streamlining your network analysis.
...
->[!NOTE]
->Azure Stack HCI and Azure Stack Hub customers should expect the Vibranium functionality.
-
-## Limitations
-
-Below are some of Packet Monitor's most significant limitations.
-
-- Only Ethernet traffic is supported for now. Wireless traffic support will be added in later releases.
-- Packet drops from Windows Firewall are not visible through Packet Monitor yet. 
+- Packet monitoring and counting at multiple locations along networking stack
```
Comment (illfated) [inline WindowsServerDocs/networking/technologies/pktmon/pktmon.md:53]:
```suggestion
- Packet monitoring and counting at multiple locations along the networking stack
```
common usage / grammar and readability

File: WindowsServerDocs/networking/technologies/pktmon/pktmon.md [hunk 2]
```diff
@@ -50,29 +50,14 @@ Use these best practices for streamlining your network analysis.
...
+- Packets drop detection at multiple stack locations
+- Flexible runtime packet filtering with encapsulation support
+- General logging and tracing support (ETW and WPP events)
+- TXT log analysis based on TcpDump packet parsing
+- Multiple logging modes: real-time, high volume in-memory, multi-file, circular
+- Ethernet, Wifi and Mobile broadband media type support
```
Comment (illfated) [inline WindowsServerDocs/networking/technologies/pktmon/pktmon.md:59]: Microsoft Docs naming conventions: Wi-Fi or WiFi (type of technology)
```suggestion
- Ethernet, Wi-Fi and Mobile broadband media type support
```

**Comment by illfated** [comment]
Thank you for creating the PR to improve the docs. I hope you find the suggestions useful.

**Comment by PRMerger20** [comment]
@george-guirguis : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger14** [comment]
@george-guirguis : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by illfated — APPROVED** [review]
Looks good to me.

**Rich Diff view** (for easier layout review):
- https://github.com/MicrosoftDocs/windowsserverdocs/pull/5479/files?short_path=50d7c48

Normal source file view:
- https://github.com/MicrosoftDocs/windowsserverdocs/pull/5479/files

**Comment by illfated** [comment]
#sign-off

**Comment by PRMerger10** [comment]
@illfated: I'm sorry - only the author of this article, @khdownie, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

**Comment by illfated** [comment]
Yes, I know. My sign-off comment is based on a previous agreement with the team.

**Review by khdownie — APPROVED** [review]

**Comment by khdownie** [comment]
#sign-off

**Comment by khdownie** [comment]
#sign-off

**Comment by khdownie** [comment]
Making these changes in a separate PR - #hold-off

**Comment by george-guirguis** [comment]
Deleting branch since @khdownie already took care of the changes in a separate PR

**Pull request closed without merging**